stable/redis/values.yaml

```diff
@@ -27,7 +27,7 @@ metrics:
   enabled: false
   # resources: {}
   # podAnnotations: {}
-  # targetServiceAnnotations:
+  # annotations:
```

Comment:
Do not change this value! `targetServiceAnnotations` is still being used.

Reply:
Ok, I will change it to `targetServiceAnnotations`.